tests/test_lab_process_list.py

```diff
@@ -44,6 +44,16 @@ def test_process_list(cli_runner: CliRunner):
         ).isoformat(),
         "status": ILAB_PROCESS_STATUS.RUNNING,
     }
+    process_registry["63866b92-799a-42a7-af02-d0b68fddf19d"] = {
```

Comment:
please generate a uuid so that it's easier to distinguish them when reading the code. one character difference is hard to spot / easy to mix.